tests/test_launch_errors.py

```diff
@@ -31,15 +31,15 @@ def test_model_failure(fileutils):
     assert all([stat == status.STATUS_FAILED for stat in statuses])
 
 
-def test_orchestrator_relaunch(fileutils):
+def test_orchestrator_relaunch(fileutils, wlmutils):
     """Test error when users try to launch second orchestrator"""
     exp_name = "test-orc-error-on-relaunch"
     exp = Experiment(exp_name, launcher="local")
     test_dir = fileutils.make_test_dir()
 
-    orc = Orchestrator(port=6780)
+    orc = Orchestrator(port=wlmutils.get_test_port())
     orc.set_path(test_dir)
-    orc_1 = Orchestrator(port=6790)
+    orc_1 = Orchestrator(port=wlmutils.get_test_port())
```

Comment:
Here we should use a different port for the second `Orchestrator`, to make sure it fails for the right reason (i.e. it fails because we do not allow for two `Orchestrator`s to be launched, not because the port is already taken). I would just add 1 to the port returned by `get_test_port()` and maybe we need to make an explicit reference to this in the test docs (or in the test headers).